content/en/docs/components/katib/hyperparameter.md

```diff
@@ -37,15 +37,32 @@ of Katib, you can use one of the following Katib installs. To install the specif
 Katib release (e.g. `v0.11.1`), modify `ref=master` to `ref=v0.11.1`.
 
 1. **Katib Standalone Installation**
-
-   Run the following command to deploy Katib with the main components
-   (`katib-controller`, `katib-ui`, `katib-mysql`, `katib-db-manager`, and `katib-cert-generator`):
-
-   ```shell
-   kubectl apply -k "github.com/kubeflow/katib.git/manifests/v1beta1/installs/katib-standalone?ref=master"
-   ```
-
-   This installation doesn't require any additional setup on your Kubernetes cluster.
+   
+   There are two ways to install Katib by standalone, 
+   both of which do not require any additional setup on your Kubernetes cluster.
+
+   1. **Basic Installation**
+
+      Run the following command to deploy Katib with the main components
+      (`katib-controller`, `katib-ui`, `katib-mysql`, `katib-db-manager`, and `katib-cert-generator`):
+
+      ```shell
+      kubectl apply -k "github.com/kubeflow/katib.git/manifests/v1beta1/installs/katib-standalone?ref=master"
+      ```
+
+   2. **Controller Leader Election Support**
+    
+      Run the following command to deploy Katib with Controller
+      [Leader Election](https://kubernetes.io/blog/2016/01/simple-leader-election-with-kubernetes/):
+
+      ```shell
+      kubectl apply -k "github.com/kubeflow/katib.git/manifests/v1beta1/installs/katib-leader-election?ref=master"
+      ```
+      
+      This installation is almost the same as `Basic Installation`,
+      although you can make katib-controller High Availability(HA) using leader election.
+      If you plan to use Katib in an environment where high SLAs and SLOs are required, 
```

Comment:
Let's not use short form for `SLOs` and `SLAs` here, since we don't mentioned about it before.

Reply:
It makes sense.